ctru-rs/src/smdh.rs

```diff
@@ -0,0 +1,99 @@
+use crate::services::cfgu::Language;
+use bitflags::bitflags;
+
+#[derive(Copy, Clone, Debug)]
+#[repr(C)]
+pub struct Smdh {
+    magic: [u8; 4],
+    version: u16,
+    titles: [SmdhTitle; 16],
+    age_ratings: [u8; 16],
```

Comment:
Age rating might be deserving of its own struct / bitflags, from reading the docs about it